### PR TITLE
Created image upload form and ImageFormServlet to handle POST request.

### PR DESCRIPTION
**Pull request by Corramel**

### CODE REVIEW

File: src/main/java/com/google/sps/servlets/ImageFormServlet.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.MultipartConfig;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.Part;
+
+@WebServlet("/upload")
```
Comment (matty377) [inline src/main/java/com/google/sps/servlets/ImageFormServlet.java:15]: This is good but another approach to consider is making the servlet more general and creating CRUD/REST operations to handle each of the different types. So you'd have a `doPost` for the upload and a `doGet` for the list.

I think this current approach is good with 4 people working on the same codebase though so I'm actually going to suggest leaving it as-is and letting the list servlet do its own thing separately.

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -10,10 +10,20 @@
...
+      <p>Click <a href="/hello">here</a> to see content created by server code.</p> <!--Could probably make this into a link of galleries of images/messages (MVP)
+                                                                                        Or when login is created, to their uploaded images page-->
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+      <div id="image-form">
+          <form action="/upload" method="POST" enctype="multipart/form-data"> <!--Redirect to page with images?-->
+                <p>Add an image and a message!</p>
+                <textarea name="message">Type a message.</textarea>
+                <br/>
```
Comment (matty377) [inline src/main/webapp/index.html:22]: Break tags are kinda hacky and not really used much any more. I'd consider a CSS flex layout on the outer div to move this stuff around.

**Review by matty377 — APPROVED** [review]
LGTM, you'll just need to find 1 more reviewer.

File: src/main/webapp/index.html (hunk 2 above)
Comment (Corramel) [reply src/main/webapp/index.html:22]: Alright, cool. I was planning to do more CSS work too. I assume it would also be easier to change later on?

**Review by InfinityGrip — APPROVED** [review]
nice job!

**Comment by matty377** [comment]
Not sure why 2 approvals isn't greening up the merge button but I'm just gonna merge it.

**Pull request merged**